commcare_connect/templates/tailwind/layouts/sidenav.html

```diff
@@ -0,0 +1,57 @@
+{% load static i18n %}
+
+<div x-data="{ sidebarOpen: false }"
+    @click.outside="sidebarOpen = false"
+    class="fixed top-0 left-0 h-screen bg-brand-deep-purple pt-3 z-40 text-white flex flex-col justify-between transition-all duration-300 px-2"
+    :class="sidebarOpen ? 'w-64' : 'w-16'">
+
+    <!-- Top Section -->
+    <div>
+        <!-- Logo Section -->
+        <div class="">
+            <div class="flex p-2 justify-start">
+                <div
+                    x-show.immediate="!sidebarOpen"
+                    class=" h-12 transition-none"
+                >
+                    <img src="{% static 'images/logo.svg' %}" alt="Compact Logo" class="w-8 h-8">
+                </div>
+                <div
+                    x-show.immediate="sidebarOpen"
+                    class="h-12 pl-1"
+                >
+                    <img src="{% static 'images/logo-full.svg' %}" alt="Full Logo" class="h-12 w-auto">
+                </div>
+            </div>
+        </div>
+
+        <div class="flex rounded-lg items-center p-2 transition-all duration-300 cursor-pointer">
+            <div class="flex items-center justify-center w-8 h-8">
+                <i class="fa-light text-lg text-brand-sky"
+                    :class="sidebarOpen ? 'fa-xmark-large': 'fa-bars'"
+                    @click="sidebarOpen = !sidebarOpen"></i>
+
+            </div>
+
+            <div class="transition-all duration-300 overflow-hidden flex items-center"
+                 :class="sidebarOpen ? 'w-40 ml-4 opacity-100' : 'w-0 opacity-0'">
+                <span class="text-xs whitespace-nowrap">Menu</span>
+            </div>
+        </div>
+
+        <!-- Navigation Items -->
+        <div class="mt-5">
+            {% url 'program:home' request.org.slug as program_home_url %}
+            {% include "tailwind/components/sidenav-items.html" with name='Programs' icon='table-columns'  target=program_home_url %}
+            {% include "tailwind/components/sidenav-items.html" with name='Opportunities' icon='table-cells-large'  target='/opportunities' %}
+            {% include "tailwind/components/sidenav-items.html" with name='Analytics' icon='chart-pie'  target='/analytics' %}
```

Comment:
I thought we were skipping this one?

Reply:
I didn't get a confirmation yet on [this](https://dimagi.slack.com/archives/C08KFKP80QM/p1744287855847319). I can remove it once Rama confirms